.changeset/fast-turtles-compare.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
All works great in storybook 🙌 I'm only wondering if changeset is needed in this case

Reply:
Good question. I'm not 100% sure as well. 
@commercetools/shield-team-fe, what do you guys think?

Reply:
I think this change won't be included in the next release if we don't add the changeset so perhaps the [public Storybook site](https://uikit.commercetools.com/) won't be updated neither (not sure about this).